src/app/feature/user/pages/user-debug/user-debug.page.html

```diff
@@ -0,0 +1,64 @@
+<ion-content class="ion-padding">
+  <div style="display: flex; align-items: center; gap: 8px">
+    <span>App User ID:</span>
+    <ion-input [(ngModel)]="importUserId" style="flex: 1"></ion-input>
+    <ion-button (click)="importUserData()" [disabled]="importUserId===userId">Import</ion-button>
```

Comment:
I'm not sure we necessarily want to disable when the IDs match – we might want to overwrite the local user data with that from when the same user last synced? I'm not sure what the real world use case would be, but as this is a debug page that might be worth allowing as it's a possible use of the `user: import` action (that's how I tested the functionality on #2270, see [comment](https://github.com/IDEMSInternational/parenting-app-ui/pull/2270#pullrequestreview-1980363795)).

Reply:
Good idea, resolved in d8fb88fd627db8f7255fa0165104b5df70a6c47f